packages/manager/modules/pci/src/projects/project/storages/databases/database/current-queries/current-queries.html

```diff
@@ -0,0 +1,127 @@
+                    class="d-inline-block"
+                    name="idle-connections-switch"
+                    model="$ctrl.showIdleConnections"
+                    on-change="$ctrl.onShowIdleConnectionsChange(modelValue)"
+                >
+                </oui-switch>
+                <label
+                    class="d-inline-block mx-2"
+                    data-ng-bind="($ctrl.showIdleConnections ? 'pci_databases_current_queries_switch_enable' : 'pci_databases_current_queries_switch_disable') | translate"
+                >
+                </label>
+            </oui-field>
+        </section>
+        <section class="col-sm-12 col-md-3">
+            <h6
+                class="my-0"
+                data-translate="pci_databases_current_queries_active_connections_label"
+            ></h6>
+            <oui-field>
+                <oui-switch
+                    class="d-inline-block"
+                    name="idle-connections-switch"
+                    model="$ctrl.showActiveConnections"
+                    on-change="$ctrl.onShowActiveConnectionsChange(modelValue)"
+                >
+                </oui-switch>
+                <label
+                    class="d-inline-block mx-2"
+                    data-ng-bind="($ctrl.showActiveConnections ? 'pci_databases_current_queries_switch_enable' : 'pci_databases_current_queries_switch_disable') | translate"
+                >
+                </label>
+            </oui-field>
+        </section>
+        <section class="col-sm-12 col-md-3">
+            <h6
+                class="my-0"
+                data-translate="pci_databases_current_queries_refresh_label"
+            ></h6>
+            <oui-field>
+                <oui-switch
+                    class="d-inline-block"
+                    name="idle-connections-switch"
+                    model="$ctrl.autorefreshPage"
+                    on-change="$ctrl.onPageAutoRefreshChange(modelValue)"
+                >
+                </oui-switch>
+                <label
+                    class="d-inline-block mx-2"
+                    data-ng-bind="($ctrl.autorefreshPage ? 'pci_databases_current_queries_switch_enable' : 'pci_databases_current_queries_switch_disable') | translate"
+                >
+                </label>
+            </oui-field>
+        </section>
+    </div>
+    <oui-datagrid
+        id="currentQueriesDataGrid"
+        class="current-queries-grid"
+        rows="$ctrl.displayQueries"
+        empty-placeholder="{{:: 'pci_databases_current_queries_datagrid_empty_placeholder' | translate }}"
+    >
+        <oui-datagrid-column
+            data-title="::'pci_databases_current_queries_datagrid_column_query' | translate"
+            property="query"
+            type="string"
+            data-sortable
+            data-searchable
+        >
+            <text-accordion
+                class="query-column"
+                value="$value"
+            ></text-accordion>
```

Comment:
Request with IDLE status have an empty query value, which makes the display a bit confusing.
Should we display something here for the user? Or at least show a column with the status ?